.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
 before_install:
 - sudo snap install microk8s --classic --channel=1.11/stable
 - curl -Lo $GOPATH/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.11.4/bin/linux/amd64/kubectl
-- curl -Lo $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64
+- curl -Lo $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64
```

Comment:
It might make sense to check if $GOBIN is set and points to a different location

Reply:
Good call, but let's do it in another PR; the dep version change is already basically a drive-by just to work around a dep bug.